samples/client/petstore/spring-cloud-async/src/main/java/org/openapitools/model/Pet.java

```diff
@@ -85,6 +85,21 @@ public static StatusEnum fromValue(String value) {
   @JsonProperty("status")
   private StatusEnum status;
 
+  /**
+   * Default constructor
+   * @deprecated Use {@link Pet#Pet(String, List<String>)}
+   */
+  @Deprecated(since = "6.5.0")
+  public Pet() {}
+
+  /**
+   * Constructor with only required parameters
+   */
+  public Pet(String name, List<String> photoUrls) {
```

Comment:
It might be good to validate that this does not interfere with Jackson mappings or the JSON with a unit test

Reply:
I'm sorry, I don't know what you mean :/

Reply:
I mean it would be good to test if the resulting DTO can be serialized and deserialized to/from JSON using a Jackson parser, to make sure the changing constructor doesn't interfere with Jackson behavior.

Reply:
Do you know if there some exemple in the source code to test that with generated code ?

Reply:
@alucas in some generated samples there are test classes which we wrote manually to test some specific things. Those test are added to `.openapi-generator-ignore` and won't be re-generated. I haven't found any sample for Spring, but this is example for java jersey2. Hope this will help


1. `.openapi-generator-ignore` which contains path to java test - https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/java/jersey2-java8/.openapi-generator-ignore
2. `JSONComposedSchemaTest.java` custom test itself - https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/java/jersey2-java8/src/test/java/org/openapitools/client/JSONComposedSchemaTest.java

With this approach you can add test cases for some spring samples. Please add to at least spring-boot & spring-boot-3 with & without inheritance. Thanks

Reply:
@alucas in some generated samples there are test classes which we wrote manually to test some specific things. Those test are added to .openapi-generator-ignore and won't be re-generated. I haven't found any sample for Spring, but this is example for java jersey2. Hope this will help

1. `.openapi-generator-ignore` which contains path to java test - https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/java/jersey2-java8/.openapi-generator-ignore
2. `JSONComposedSchemaTest.java` custom test itself - https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/java/jersey2-java8/src/test/java/org/openapitools/client/JSONComposedSchemaTest.java

With this approach you can add test cases for some spring samples. Please add to at least spring-boot & spring-boot-3 with & without inheritance. Thanks

Reply:
done